docs/run-a-node/setup/install.md

```diff
@@ -26,6 +26,26 @@ Due to the higher TPS on MainNet, to successfully run an Algorand MainNet node,
 
 Participation nodes (especially those with high stake) and relays have higher requirements to ensure the performance of the overall blockchain.
 
+Recommended system specification for participation nodes is:
+
+* 8 vCPU
+* 16 GB RAM
+* 100 GB NVMe SSD or equivalent
+* 1 Gbps connection with low latency
+
+Recommended system specification for relay nodes is:
+
+* 16 vCPU
+* 32 GB RAM
+* 3 TB NVMe SSD or equivalent
+* 30 TB/month egress
+* 1 Gbps connection with very low latency
+
+The third-party website [Algoscan Analytics](https://developer.algoscan.app/) indicates the current size of the data folder for MainNet/TestNet/BetaNet archival nodes.
+
+!!! info
+    Private networks used for development require must lower specs as they are usually achieving much lower TPS. A Raspberry Pi with 2GB of RAM is sufficient for low-TPS private networks.
```

Comment:
@fabrice102 Looks like there _might_ be a typo.  I left a _suggestion_ with what I believe to be the intent.

Feel welcomed to disregard.  I looked at the PR out of personal interest to review hardware specs.

```suggestion
    Private networks used for development require much lower specs as they are usually achieving much lower TPS. A Raspberry Pi with 2GB of RAM is sufficient for low-TPS private networks.
```